pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -676,3 +682,21 @@ message RemoveImageRequest {
 }
 
 message RemoveImageResponse {}
+
+enum ConfigKey {
+    POD_CIDR = 1;
+    HAIRPIN_MODE = 2;
+    NON_MASQUERADE_CIDRS = 3;
+    NETWORK_PLUGIN_DIR = 4;
+}
+
+message RuntimeConfig {
```

Comment:
Is this `NetworkConfig` or a more generic `RuntimeConfig`? i think `PushRuntimeConfig` should be a general purpose call that can be used to update a single config object (not restricted to network).